.github/workflows/action-conda-publish.yml

```
@@ -0,0 +1,78 @@
+name: Anaconda Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  publish:
+    name: publish / python-3.8 / ubuntu-latest
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: goanpeca/setup-miniconda@v1
+        with:
+          activate-environment: esmvaltool
+          auto-update-conda: true
+          environment-file: environment.yml
+          python-version: "3.8"
+      - name: Show conda config
+        shell: bash -l {0}
+        run: |
+          conda info
+          conda list
+          conda config --show-sources
+          conda config --show
+      - name: Python info
+        shell: bash -l {0}
+        run: |
+          which python
+          python --version
+      - name: Install build dependencies
+        shell: bash -l {0}
+        run: conda install -y conda-build conda-verify ripgrep anaconda-client
+      - name: Build the conda package
+        shell: bash -l {0}
+        run: |
+          conda config --set anaconda_upload no
+          export BUILD_FOLDER=/tmp/esmvalcore/_build
+          mkdir -p $BUILD_FOLDER
+          conda build package \
+            --channel esmvalgroup --channel conda-forge \
+            --croot $BUILD_FOLDER \
+      - name: Push the package to anaconda cloud
+        shell: bash -l {0}
+        run: |
+          export BUILD_FOLDER=/tmp/esmvalcore/_build
+          anaconda --token ${{ secrets.ANACONDA_TOKEN }} upload --user esmvalgroup --force $BUILD_FOLDER/noarch/*.tar.bz2
```

```suggestion
          anaconda --token ${{ secrets.ANACONDA_TOKEN }} upload --user esmvalgroup $BUILD_FOLDER/noarch/*.tar.bz2
```
I'm not sure if we want to force uploading, because we do not want to upload if there are errors